Core/GDCore/IDE/Events/EventsRefactorer.h

```diff
@@ -146,6 +147,19 @@ class GD_CORE_API EventsRefactorer {
                                        gd::InstructionsList& instructions,
                                        gd::String oldName,
                                        gd::String newName);
+   /**
+   * Replace all occurences of an object name by another name in a parameter of event.

```

Comment:
Actually this function is doing a job not tied to an event:

`Replace all occurrences of an object name by another name in an expression with the specified metadata.`